grafana/overlays/nerc-ocp-obs/grafanas/grafana.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: integreatly.org/v1alpha1
+kind: Grafana
+metadata:
+  name: grafana
+  namespace: grafana
+spec:
+  deployment:
+    envFrom:
+      - secretRef:
+          name: oauth-client-secret
+      - secretRef:
+          name: grafana-serviceaccount-token
+      - configMapRef:
+          name: openshift-service-ca.crt
+  config:
+    server:
+      root_url: https://grafana.apps.nerc-ocp-obs.rc.fas.harvard.edu
+    auth.generic_oauth:
+      enabled: true
+      scopes: openid email groups profile
+      email_attribute_path: name
+      api_url: https://dex-dex.apps.nerc-ocp-infra.rc.fas.harvard.edu/userinfo
```

Comment:
This is all configuring who can log into to grafana and the observability cluster, right? So we should be authenticating against the observability cluster, not against the infra cluster (because people will have access to the obs cluster who should not have access to the infra cluster).